pkg/build/builder/docker_test.go

```diff
@@ -192,6 +192,14 @@ func TestDockerfilePath(t *testing.T) {
 		"\"io.openshift.build.commit.message\"=\"message\"",
 	}
 
+	if isNew, err := util.CreateBaseDir(util.GetBaseDir()); err == nil {
```

Comment:
Unit tests should not be creating output into BASETMPDIR - they should simply use the go tempdir function